src/components/OfflineWithFeedback.tsx

```diff
@@ -83,8 +78,12 @@ function OfflineWithFeedback({
     const {isOffline} = useNetwork();
 
     const hasErrors = isNotEmptyObject(errors ?? {});
+
     // Some errors have a null message. This is used to apply opacity only and to avoid showing redundant messages.
-    const errorMessages = omitBy(errors, (e) => e === null);
+    const errorEntries = Object.entries(errors ?? {});
+    const filteredErrorEntries = errorEntries.filter((errorEntry): errorEntry is [string, string] => errorEntry[1] !== null);
+    const errorMessages = Object.fromEntries(filteredErrorEntries);
```

Comment:
The reason why it was changed is that errors can have a null message (you can even see it in the comment above). Updated code narrow down the types to identify that updated `errorMessages` object doesn't have null values